fdm-app/app/routes/farm.$b_id_farm.$calendar.field.fertilizer.tsx

```diff
@@ -0,0 +1,469 @@
+import {
+    addFertilizerApplication,
+    getFarms,
+    getFertilizerParametersDescription,
+    getFertilizers,
+    getFields,
+} from "@svenvw/fdm-core"
+import {
+    data,
+    type ActionFunctionArgs,
+    type LoaderFunctionArgs,
+    type MetaFunction,
+    NavLink,
+    redirect,
+    useLoaderData,
+    useLocation,
+    useNavigation,
+    useSearchParams,
+} from "react-router"
+import { FarmTitle } from "~/components/blocks/farm/farm-title"
+import { Header } from "~/components/blocks/header/base"
+import { HeaderFarm } from "~/components/blocks/header/farm"
+import { Button } from "~/components/ui/button"
+import { SidebarInset } from "~/components/ui/sidebar"
+import { getSession } from "~/lib/auth.server"
+import { getCalendar, getTimeframe } from "~/lib/calendar"
+import { clientConfig } from "~/lib/config"
+import { handleActionError, handleLoaderError } from "~/lib/error"
+import { fdm } from "~/lib/fdm.server"
+import { FarmContent } from "~/components/blocks/farm/farm-content"
+import {
+    BreadcrumbItem,
+    BreadcrumbSeparator,
+} from "../components/ui/breadcrumb"
+import { FertilizerApplicationForm } from "~/components/blocks/fertilizer-applications/form"
```

Comment:
both "../" and "~/" works?

Reply:
Yes, in this case they point to the same location. However, `~/` is preferred. I will change that here